kolibri/plugins/coach/assets/src/views/exams-page/preview-exam-modal.vue

```diff
@@ -1,7 +1,40 @@
 <template>
 
-  <core-modal :title="`${examTitle} ${$tr('preview')}`" @cancel="close">
-    <div>Exam Preview</div>
+  <core-modal :title="$tr('preview')" @cancel="close" maxWidth="100%">
+    <div class="question-selector-container">
+      <div class="question-selector">
+        <div>
+          {{ $tr('numQuestions', { num: exam.question_count })}}<icon-button :text="$tr('randomize')" @click="$emit('randomize')" :primary="true"/>
+        </div>
+        <ui-collapsible v-for="(source, exIndex) in questionSources" :title="$tr('exercise', { num: exIndex + 1 })" :open="exIndex===0">
```

Comment:
minor: 

 * recommend long lists of attributes on multiple lines
 * spaces around operators, e.g. `exIndex ===  0`